src/rpc/server.h

```diff
@@ -45,8 +47,9 @@ class JSONRPCRequest
     bool fHelp;
     std::string URI;
     std::string authUser;
+    CWallet *wallet;
```

Comment:
Do we want wallet here even an opaque pointer?

Reply:
? This is where it belongs..